packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx

```diff
@@ -38,6 +37,14 @@ type DangerZoneProps = {
 
 const DATE_FORMAT = 'yyyy-mm-dd @ HH:MM:ss Z';
 
+const styleClasses = {
+  enabledOrVerified: 'font-semibold text-green-900',
+  notEnabledOrVerified: 'font-semibold text-red-600',
+  gradientInfoDisplay: 'border-l-2 border-grey-500 mb-8 pl-4',
```

Comment:
I think since this is no longer a gradient we could call `gradientInfoDisplay` something else 🤔

It also still sorta feels like we could possibly use `@apply` for `.account li`, but since we don't have any other custom SCSS now in the admin panel after this PR, it also feels fine to leave it how you have it.